fitbenchmarking/utils/exceptions.py

```diff
@@ -0,0 +1,127 @@
+"""
+This file holds all FitBenchmarking exceptions, organised by exception id
+"""
+
+
+class FitBenchmarkException(Exception):
```

Comment:
All looks good but is it possible to test this file too?

Reply:
I'm not sure how to test this file... Ideally I think this would be tested as a by-product of testing the files that we expect to raise them, although that would be a separate task.